src/main/fc/tasks.c

```diff
@@ -368,7 +368,7 @@ task_attribute_t task_attributes[TASK_COUNT] = {
     [TASK_ATTITUDE] = DEFINE_TASK("ATTITUDE", NULL, NULL, imuUpdateAttitude, TASK_PERIOD_HZ(100), TASK_PRIORITY_MEDIUM),
 #endif
 
-    [TASK_RX] = DEFINE_TASK("RX", NULL, rxUpdateCheck, taskUpdateRxMain, TASK_PERIOD_HZ(33), TASK_PRIORITY_HIGH), // If event-based scheduling doesn't work, fallback to periodic scheduling
+    [TASK_RX] = DEFINE_TASK("RX", NULL, rxUpdateCheck, taskUpdateRxMain, TASK_PERIOD_HZ(22), TASK_PRIORITY_HIGH), // If event-based scheduling doesn't work, fallback to periodic scheduling
```

Comment:
This has no impact as the minimum task rate is set by the rate at which `rxUpdateCheck()` returns `true` which, with no rx connected (or sending valid data), is determined by `rxDataProcessingRequired` being set `true` as a consequence of `needRxSignalBefore` expiring as determined by the value of `needRxSignalMaxDelayUs`, `DELAY_100_MS`. The value of `DELAY_100_MS` results in a min task update rate of 10Hz as shown below.

```
# tasks
Task list             rate/hz  max/us  avg/us maxload avgload  total/ms   late    run reqd/us
00 - (         SYSTEM)     10       1       0    0.0%    0.0%         0      0     10       1
01 - (         SYSTEM)    986       1       0    0.0%    0.0%        12      0   1008       0
02 - (           GYRO)   7937      11       8    8.7%    6.5%      1831      0   8059       0
03 - (         FILTER)   7937      14      10   11.1%    7.9%      2181      0   8059       0
04 - (            PID)   7937      33      24   26.1%   19.2%      5567      0   8059       0
05 - (            ACC)    985      13       9    1.2%    0.9%       263      0   1008      11
06 - (       ATTITUDE)    100       9       6    0.0%    0.0%        17      0    103       8
07 - (             RX)     10      16      10    0.0%    0.0%         9      0     30       8
```

Reply:
@SteveCEvans - thanks for noting this issue.  Happy to hold off until your concerns are addressed.

The *intent* in this PR is that, after receiving valid data, there is a wait for `RXLOSS_TRIGGER_INTERVAL` of 150ms, for some new data to arrive.  After that, if still no data, we then regularly re-check at 50ms intervals for a data packet.  Otherwise we handle new values as soon as they arrive.

With this PR, when I power up the FC, with no Rx link, tasks say:
```
7 - (             RX)     20      32      21    0.0%    0.0%        44      1    567      21
0
```

It also does the same if I switch the radio off.

So it seems that with this PR it is following the new 50ms re-check interval, resulting in 20Hz checks.

<aside> - I didn't realise we can use the RX task to indicate the frequency with which we. check the Rx packets. It's very cool to see 500Hz exactly when I am connected with a link speed of 500Hz!  :-) 

I am not as expert as you are in the management of the tasks themselves, and would be grateful if you could either approve this PR and make a separate PR to update the tasks, if needed.  

Alternatively, let me know what should be done at tasks level, and I can do it here, if it's not too complicated.

Let me know how best to proceed - thanks!